<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFrontWindow</class>
 <widget class="QDialog" name="QFrontWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiClass</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>335</y>
     <width>185</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>292</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 112, 112)</string>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QWifiIcon" name="wifi_icon" native="true">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>45</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start_or_stop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>160</width>
     <height>76</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(112, 112, 112)</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>285</y>
     <width>185</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="brow" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>64</width>
     <height>34</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>brow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>exit</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_nextpage">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>496</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWifiIcon</class>
   <extends>QWidget</extends>
   <header>QWifiIcon.h</header>
   <container>1</container>
   <slots>
    <slot>setStyle()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>button_start_or_stop</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>button_exit</tabstop>
  <tabstop>button_nextpage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_exit</sender>
   <signal>clicked()</signal>
   <receiver>QFrontWindow</receiver>
   <slot>onBtnExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_nextpage</sender>
   <signal>clicked()</signal>
   <receiver>QFrontWindow</receiver>
   <slot>onBtnRotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_start_or_stop</sender>
   <signal>clicked()</signal>
   <receiver>QFrontWindow</receiver>
   <slot>onBtnStartOrStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_nextpage</sender>
   <signal>clicked()</signal>
   <receiver>QFrontWindow</receiver>
   <slot>onTurnNextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnRotate()</slot>
  <slot>onBtnExit()</slot>
  <slot>onBtnStartOrStop()</slot>
  <slot>onTurnNextPage()</slot>
 </slots>
</ui>
